data-prepper-plugins/aggregate-processor/README.md

```diff
@@ -75,6 +76,23 @@ While not necessary, a great way to set up the Aggregate Processor [identificati
         { "sourceIp": "127.0.0.1", "destinationIp": "192.168.0.1", "status": 200, "bytes": 1000, "http_verb": "GET" }
       ```
 
+### <a name="count"></a>
+* `count`: Count Events belonging to the same group and generate a new event with values of the identification keys and the count, indicating the number of events. All Events that make up the combined Event will be dropped.
```

Comment:
Can we update this section with the full configuration for `output_format` and `count_key` and result examples for default output in otel metrics. Other than that I am on board to approve